.eslintignore

```diff
@@ -1,3 +1,3 @@
-node_modules
-dist
+node_modules/
+dist/
 app/utils/BFS.js
```

Comment:
Remove. Replace with a comment inside the file itself.

Reply:
done